src/NUnitEngine/nunit.engine.netstandard/nunit.engine.netstandard.csproj

```diff
@@ -12,6 +12,11 @@
     <AssemblyOriginatorKeyFile>..\..\nunit.snk</AssemblyOriginatorKeyFile>
     <GeneratePackageOnBuild>True</GeneratePackageOnBuild>
     <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
+    <PackageId>nunit.engine.netstandard</PackageId>
+    <Title>NUnit .NET Standard Engine</Title>
```

Comment:
Suggestion - we have problems with naming lots of similar NuGet packages. This is equivalent to the package we call "NUnit Test Engine" - correct? How about "NUnit Test Engine - .NET Standard" or something, to reflect that these are related?